CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## 2.2.0
```

Comment:
I know this PR didn't add this line, but in the future, I think it would be best to leave unreleased changes under an "unreleased" section. Currently it looks like a 2.2.0 release exists and we're adding functionality to that existing release; pretty weird.